src/main/java/org/cyclonedx/maven/BaseCycloneDxMojo.java

```diff
@@ -612,7 +612,7 @@ private void extractMetadata(MavenProject project, Component component) {
         if (CycloneDxSchema.Version.VERSION_10 != schemaVersion()) {
             if (project.getOrganization() != null && project.getOrganization().getUrl() != null) {
```

Comment:
If we're getting the URL from the project, and not the organization, then I think this if statement should be removed?

Reply:
you're right, I went too fast :)